### PR TITLE
Starred_Messages_Feature_Contd_II/Outreachy

### PR DESCRIPTION
**Pull request by yaya-usman**
Task:  [vector-im/element-web#22453](https://github.com/vector-im/element-web/issues/22453)

Signed-off-by: Yaya Usman [yaya-usman@users.noreply.github.com](mailto:yaya-usman@users.noreply.github.com) 

Type: enhancement

---
Here's what your changelog entry will look like:

## ✨ Features
 * Starred_Messages_Feature_Contd_II/Outreachy ([\#9086](https://github.com/matrix-org/matrix-react-sdk/pull/9086)).

### CODE REVIEW

**Review by andybalaam — CHANGES_REQUESTED** [review]
Looks great - just a couple of things to fix!

File: src/components/views/rooms/RoomList.tsx [hunk 1]
```diff
@@ -351,6 +353,11 @@ const TAG_AESTHETICS: ITagAestheticsMap = {
         isInvite: false,
         defaultHidden: false,
     },
+    [DefaultTagID.SavedItems]: {
+        sectionLabel: _td("Saved Items"),
```
[the remark this answers is not in the document]
Comment (yaya-usman) [reply src/components/views/rooms/RoomList.tsx:357]: it's already done sir

[the remark this answers is not in the document]
Comment (yaya-usman) [reply src/components/views/rooms/RoomList.tsx:357]: It was not having any translation issues when i previewed it to you morning

[the remark this answers is not in the document]
Comment (andybalaam) [reply src/components/views/rooms/RoomList.tsx:357]: Ah sorry, you've done it - well done.

**Review by andybalaam — APPROVED** [review]
All good, merge when ready!

**Pull request merged**